### DISCUSSION TITLE
Always get a SyntaxError in "parse" when importing stumpy

### DISCUSSION THREAD

**Discussion by DirkH79** [TDAmeritrade/stumpy#75] [Q&A]
Hi, I'm new here and just started working with python. When looking for some powerful data analysis tools for time series data I became aware of stumpy. After installation via conda, I wanted to import it, but I'm always getting a SyntaxError (see output below).
I'm using python 3.6.

Can anyone help?
Many thanks in advance.
Dirk

import stumpy
Traceback (most recent call last):

  File "C:\ProgramData\Anaconda3\lib\site-packages\IPython\core\interactiveshell.py", line 3296, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-1-e870883bf2eb>", line 1, in <module>
    import stumpy

  File "C:\ProgramData\Anaconda3\lib\site-packages\stumpy\__init__.py", line 38, in <module>
    module = ast.parse(file_contents)

  File "C:\ProgramData\Anaconda3\lib\ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)

  File "<unknown>", line 29
    Î£_T,
       ^
SyntaxError: invalid character in identifier

**Comment by seanlaw** [comment]
@DirkH79 Thank you for pointing this out! I can see where the problem is so let me try some things on my end to see if we can fix this.

In the meantime, can I ask if you are executing this within a fresh conda environment?

**Comment by seanlaw** [comment]
Okay, I see that tests are passing for Mac OS X and Linux but, for some reason, failing for Windows. Let me work on this and hopefully have a fix ASAP

**Comment by DirkH79** [comment]
Great! It's working now! Many thanks! ;-)

**Comment by seanlaw** [comment]
Great. For future reference as others are reading this, this was fixed in version 1.2.2 so please make sure you have an equal or newer version.

@DirkH79 Thanks for reporting this!!
